src/vscode-bicep-ui/packages/components/src/pan-zoom/PanZoomProvider.tsx

```diff
@@ -0,0 +1,7 @@
+import type { PropsWithChildren } from "react";
+
+import { Provider } from "./atoms";
+
+export function PanZoomProvider({ children }: PropsWithChildren) {
```

Comment:
Is the provider necessary for creating any PanZoom element?

Reply:
Yes, it's used to allow multiple instances of the `PanZoom` component.